spree_reviews.gemspec

```diff
@@ -9,38 +9,26 @@ Gem::Specification.new do |s|
   s.version     = SpreeReviews.version
   s.summary     = 'Basic review and ratings facility for Spree'
   s.description = s.summary
-  s.required_ruby_version = '>= 2.1.0'
+  s.required_ruby_version = '>= 2.5.0'
 
-  s.authors 	   = ['Paul Callaghan']
-  s.email        = 'paulcc.two@gmail.com'
-  s.homepage     = 'https://github.com/spree-contrib/spree-reviews/'
-  s.license      = 'BSD-3'
+  s.author    = ['Paul Callaghan']
+  s.email     = 'paulcc.two@gmail.com'
+  s.homepage  = 'https://github.com/spree-contrib/spree-reviews/'
+  s.license = 'BSD-3'
 
-  s.files        = `git ls-files`.split("\n")
-  s.test_files   = `git ls-files -- spec/*`.split("\n")
+  s.files = `git ls-files`.split("\n").reject { |f| f.match(/^spec/) && !f.match(/^spec\/fixtures/) }
   s.require_path = 'lib'
   s.requirements << 'none'
 
-  spree_version = '>= 3.1.0', '< 4.0'
-  s.add_runtime_dependency 'spree_core', spree_version
-  s.add_runtime_dependency 'spree_auth_devise', spree_version
-  s.add_runtime_dependency 'spree_extension'
+  spree_version = '>= 3.2.0', '< 5.0'
+  s.add_dependency 'spree_api', spree_version
+  s.add_dependency 'spree_backend', spree_version
+  s.add_dependency 'spree_core', spree_version
+
+  s.add_dependency 'deface', '~> 1.0'
+  s.add_dependency 'spree_auth_devise'
```

Comment:
@MatthewKennedy I think we can skip spree auth devise here, it's only for running test etc

Reply:
Sorted.